certora/specs/SafeWebAuthnSignerSingleton.spec

```diff
@@ -46,6 +54,7 @@ rule verifySignatureUniqueness(env e){
     bytes4 second_message_verified = isValidSignature(e, second_message, signature);
 
     assert (first_message != second_message) => !(first_message_verified == MAGIC_VALUE() && second_message_verified == MAGIC_VALUE());
+    satisfy true;
```

Comment:
Why do we need those satisfy true here?
It's not the same as using rule_sanity basic?

Reply:
It is, but Ive added it to have also a call trace for a valid run.